test/sagas/index_test.js

```diff
@@ -1,86 +1,60 @@
 import { expect } from "chai";
 import sinon from "sinon";
-import { fork } from "redux-saga/effects";
 
 import { SESSION_SERVERINFO_SUCCESS } from "../../scripts/constants";
-const routeSagas = require("../../scripts/sagas/route");
+// const routeSagas = require("../../scripts/sagas/route");
 const sessionSagas = require("../../scripts/sagas/session");
 const bucketSagas = require("../../scripts/sagas/bucket");
 const collectionSagas = require("../../scripts/sagas/collection");
-import rootSaga from "../../scripts/sagas";
 import configureStore from "../../scripts/store/configureStore";
 
+import * as sessionActions from "../../scripts/actions/session";
 import * as bucketActions from "../../scripts/actions/bucket";
 
 
+function expectSagaCalled(saga, action) {
+  // Note: the rootSaga function is called by configureStore
+  configureStore().dispatch(action);
+
+  expect(saga.firstCall.args[0].name).eql("bound getState");
```

Comment:
No, that's fine, "it comes from V8" is sufficient for my purposes. I thought you were hand-making a mock or something of the store, but couldn't figure out how.